modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/org.elasticsearch.txt

```diff
@@ -86,6 +86,50 @@ class org.elasticsearch.index.fielddata.ScriptDocValues$Dates {
   List getDates()
 }
 
+class org.elasticsearch.index.fielddata.ScriptDocValues$Dates {
+  org.joda.time.ReadableDateTime get(int)
+  org.joda.time.ReadableDateTime getValue()
+  List getValues()
+  org.joda.time.ReadableDateTime getDate()
+  List getDates()
+}
+
```

Comment:
the new sub class of RDT, now whitelisted and with overloads to take tz, and locale (for dayname)